CONTRIBUTING.md

```diff
@@ -15,15 +15,9 @@ Please see https://git.k8s.io/community/CLA.md for more info.
 
 ## How to build kubebuilder locally
 
-1. Build
-```sh
-$ make build
-```
-
-1. Test
-```sh
-$ ./test.sh
-```
+1. Ensure that the kubebuilder is cloned in the path `$GO_PATH/src/sigs.k8s.io/kubebuilder`
```

Comment:
This should not be necessary, since KB uses Go modules.  If it is, we've got a bug in our build scripts.

Reply:
(as long as you start with `GO111MODULE=on`)

Reply:
you probably right.